@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -52,7 +57,7 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
         // eslint-disable-next-line no-new
         new RFB(this.state.container.current, url, {
           wsProtocols: ['binary'],
-        })
+        }).addEventListener('disconnect', this.effects._connect)
```

Comment:
Were you able to test this? I think that if it gets disconnected, then it will never stop retrying to connect, even if you leave the page. And if you do that multiple times, then you'll have multiple infinite loops running in the background.

Reply:
I tested it and the behaviour was correct. As soon as the component `Console` is unmounted, the loop is stoped.
There is 2 way to unmount the component, or the `power_state !== 'Running'` or we go in another page